gconf/model.go

```diff
@@ -0,0 +1,107 @@
+package gconf
+
+import (
+	"reflect"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/orm"
+)
+
+var _ orm.CloneableData = (*Conf)(nil)
+
+func (c *Conf) Validate() error {
+	return nil
+}
+
+func (c *Conf) Copy() orm.CloneableData {
+	// TODO
+	return c
+}
+
+type ConfBucket struct {
+	orm.Bucket
+}
+
+func NewConfBucket() *ConfBucket {
+	return &ConfBucket{
+		Bucket: orm.NewBucket("gconf", orm.NewSimpleObj(nil, &Conf{})),
+	}
+}
+
+func NewConf(propName []byte, value interface{}) (orm.Object, error) {
+	var conf Conf
+
+	switch v := value.(type) {
+	case string:
+		conf.Value = &Conf_String_{String_: v}
+	case int64:
+		conf.Value = &Conf_Int{Int: v}
+	case []byte:
+		conf.Value = &Conf_Bytes{Bytes: v}
+	case weave.Address:
```

Comment:
I like this switch, along with loose checks (address and pointer/non-pointer coin)